### PR TITLE
Improving the application

### PR DESCRIPTION
**Pull request by viniblack**
- Create file .env.example
- Improving data visualization from within the table
- Correcting light mode
- Add bootstrap

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import '@/styles/globals.css'
+import "bootstrap/dist/css/bootstrap.min.css";
+import "@/styles/globals.css";
 
 export default function App({ Component, pageProps }) {
-  return <Component {...pageProps} />
```
Comment (danicuki) [inline pages/_app.js:4]: we dont use ;

File: pages/_document.js [hunk 2]
```diff
@@ -25,5 +25,5 @@ export default function Document() {
         <NextScript />
       </body>
     </Html>
```
Comment (danicuki) [inline pages/_document.js]: some here. No ;

File: pages/index.js [hunk 3]
```diff
@@ -110,25 +127,36 @@ export default function Home({ blockchains }) {
...
+const filePath = path.join(process.cwd(), "public", "blockchains.json")
   const jsonData = fs.readFileSync(filePath, "utf8")
   const blockchains = JSON.parse(jsonData)
 
   return {
     props: {
       blockchains,
     },
-  }
+  };
```
Comment (danicuki) [inline pages/index.js]: no ;

File: styles/Home.module.css [hunk 4]
```diff
@@ -86,7 +80,7 @@
 
 .center::before,
 .center::after {
-  content: '';
```
Comment (danicuki) [inline styles/Home.module.css]: why these changes?

File: styles/globals.css [hunk 5]
```diff
@@ -1,13 +1,13 @@
 :root {
   --max-width: 1100px;
   --border-radius: 12px;
-  --font-mono: ui-monospace, Menlo, Monaco, 'Cascadia Mono', 'Segoe UI Mono',
-    'Roboto Mono', 'Oxygen Mono', 'Ubuntu Monospace', 'Source Code Pro',
```
Comment (danicuki) [inline styles/globals.css:5]: why this?

**Review by danicuki — CHANGES_REQUESTED** [review]
Great contribution. fix small changes and I can accept this PR.

File: pages/index.js (hunk 3 above)
Comment (viniblack) [reply pages/index.js]: okay

File: styles/Home.module.css (hunk 4 above)
Comment (viniblack) [reply styles/Home.module.css]: My code formatter was set to use double quotes instead of single quotes. I ended up fixing this.

**Review by viniblack — COMMENTED** [review]
@danicuki i have adjusted the points in response to your comments

**Comment by danicuki** [comment]
@viniblack  there are conflicts with the main branch. Please rebase it.

**Comment by viniblack** [comment]
I think I solved the conflicts by giving git rebase, I'm not sure if it solved all the conflicts, I still have a bit of difficulty with this part of git. 😅

**Comment by danicuki** [comment]
not yet! It says: "This branch cannot be rebased due to conflicts" 
You have to rebase your repo with the

**Comment by danicuki** [comment]
You have duplicated commits on these PR. I don't know exactly what did you do. You have to make a clean PR from the last version of the main repo only with new stuff

**Comment by danicuki** [comment]
My recommendation, if you can't make it only using git is to create a fresh new fork of the main repo, change the supposed files and make a single commit with all modifications

**Pull request closed without merging**